fsmonitor.c

```diff
@@ -32,17 +33,27 @@ int read_fsmonitor_extension(struct index_state *istate, const void *data,
 	uint32_t ewah_size;
 	struct ewah_bitmap *fsmonitor_dirty;
 	int ret;
+	uint64_t tm;
```

Comment:
I would like to see `tm` renamed to `tick` or `timestamp`.

Reply:
agreed.  `tm` is usually associated with `struct tm` and this usage would be confusing.